src/tuples.py

```diff
@@ -17,28 +17,33 @@
 
 import math
 
+
 def dist(a, b):
     """Compute the distance between two x,y points."""
     x0, y0 = a  # Destructuring assignment
     x1, y1 = b
-    
+
     return math.sqrt((x1 - x0)**2 + (y1 - y0)**2)
 
+
 a = (2, 7)   # <-- x,y coordinates stored in tuples
 b = (-14, 72)
 
 # Prints "Distance is 66.94"
 print("Distance is: {:.2f}".format(dist(a, b)))
 
 
-
 # Write a function `print_tuple` that prints all the values in a tuple
 
 # YOUR CODE HERE
+def print_tuple(t):
+    for i in range(len(t)):
```

Comment:
Python's for loop is pretty smart.  You don't need to define the range if you want to traverse through an object of a defined size.  You can just do 'for i in t' then print(i).  Remember Beej talked about Python's for loop can act like forEach in javascript?